Objects/rangeobject.c

```diff
@@ -171,6 +171,28 @@ range_dealloc(rangeobject *r)
     PyObject_Free(r);
 }
 
+static unsigned long
+get_len_of_range(long lo, long hi, long step);
+
+/// Return the length as a long, or -1 on error
```

Comment:
Style: there's no convention of using triple slashes in the CPython source; suggest a double slash or old-style `/* ... */`, which would be more consistent with the rest of the module.

Reply:
Could you please clarify in the comment that a `-1` return without an exception set means overflow, while a `-1` with exception set is another exception that should be propagated as usual.